tests/environment.py

```diff
@@ -65,7 +65,8 @@ def prplmesh_command(self, command: str, *args: str) -> bytes:
         '''
```

Comment:
This commit is not related to this PR at all merge it as a separate fixup .
Also fix commit message typos.
`Commit 603b38e adds an option not to fail on log missmatch.`

Reply:
I need this commit to pass testflows on CI. Without it, it marks it as error on the test log.
But I will fix "missmatch" -> "mismatch"